### PR TITLE
exportChain code coverage

### PR DESCRIPTION
**Pull request by AmberKiso**
## Summary
Increase exportChain.ts code coverage to 100%
## Testing Plan
yarn lint
yarn test
## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Comment by AmberKiso** [comment]
Oops. Got to put this on staging first.

**Comment by AmberKiso** [comment]
And #1453 changed the request functionality. Will rework this.

File: ironfish/src/rpc/routes/chain/exportChain.test.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    value = await response.contentStream().next()
+    expect(response.status).toBe(200)
+
+    //Receive the second block
+    value = await response.contentStream().next()
+    expect(response.status).toBe(200)
+
+    //Test contents of second block. Hash seems to be regenerated differently each run.
+    expect(value).toMatchObject({
+      value: { block: { head: true, prev: genesis.header.hash.toString('hex'), seq: 2 } },
```
Comment (NullSoldier) [inline ironfish/src/rpc/routes/chain/exportChain.test.ts:40]: you should check that hash is equal to blockA1.hash

Comment (NullSoldier) [reply ironfish/src/rpc/routes/chain/exportChain.test.ts]: I made the change, it's good now.

**Review by NullSoldier — APPROVED** [review]

**Comment by NullSoldier** [comment]
Looks good, thanks for the contribution!

**Comment by AmberKiso** [comment]
Thank you!

**Pull request merged**